scss/partials/_pagination.scss

```diff
@@ -7,34 +7,35 @@
     padding-left: 0;
   }
 
-  &__list-item {
+  & &__list-item {
+    color: $secondary-blue-1;
     top: 50%;
     list-style: none;
     display: inline-block;
     margin-top: 0;
 
     &::before {
       content: "";
-      margin: 0;
-      width: 0;
+      margin-top: 0;
     }
   }
 
   &__page {
+    @extend .tertiary-info;
     @include clear-decoration();
     padding: 1.5rem 1rem;
 
-    &--active {
+    .pagination &--active {
```

Comment:
I'm a little concerned about this and some of the other selector structuring in this file since it's creating additional specificity beyond normal BEM. Don't change it right now if it's working, but we might just want to look at this down the road for optimization.

Reply:
It's a desperate move. There are a lot of fixes with this same approach all around so we definitely need to circle back on them.